source/index.txt

```diff
@@ -13,10 +13,19 @@ Introduction
 ------------
 
 Welcome to the documentation site for the official {+provider-long+}.
-You can add the provider to your .NET application as an object-relational mapper (ORM)
-to work with data in MongoDB.
+
+You can add the {+provider-short+} to your .NET application as an object-relational mapper
+(ORM) to work with data in MongoDB. The provider allows you to access and manipulate data in
+your application without explicitly running database commands. The {+provider-short+}
+tracks any queries or data changes you make, and automatically runs them on the
+database using the MongoDB Query API.
```

Comment:
Issue:
From the style guide on "[using](https://www.mongodb.com/docs/meta/style-guide/terminology/alphabetical-terms/#std-term-using)":
> Don't use using (alone) or with. By using is easier for worldwide audiences to understand.

Suggestion:
```suggestion
database by using the MongoDB Query API.
```